apps/em/em-ped-api/em-ped-api.yaml

```diff
@@ -1,16 +1,19 @@
-apiVersion: helm.fluxcd.io/v1
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
 kind: HelmRelease
 metadata:
   name: em-ped-api
 spec:
   releaseName: em-ped-api
-  chart:
-    git: git@github.com:hmcts/hmcts-charts
-    ref: master
-    path: stable/em-ped-api
   values:
     java:
       useInterpodAntiAffinity: true
       replicas: 2
       image: hmctspublic.azurecr.io/em/ped-api:prod-7d5e122-20220921222714 #{"$imagepolicy": "flux-system:em-ped-api"}
-      environment:
+  chart:
+    spec:
+      chart: em-ped-api
```

Comment:
this is probably not right.

Reply:
Just updated the v2 mapping only

Reply:
I meant the chart path is wrong, see other apps. Everything else is fine.